[2022-11-26T11:31:49.583+0000] {taskinstance.py:1155} INFO - Dependencies not met for <TaskInstance: finance_data_lake.bronze_zone scheduled__2022-11-25T11:24:47.732019+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2022-11-26T11:31:49.584+0000] {local_task_job.py:98} INFO - Task is not able to be run
